src/main/java/Awex.java

```diff
@@ -0,0 +1,130 @@
+import java.util.*;
+
+public class Awex {
+    public static void message() {
```

Comment:
Good abstraction of the message to display the list of commands available. Perhaps a better and more specific variable name could be used here to describe this function? For example you could rename it to: `listAllCommands()`